### ISSUE TITLE
note priority on release

### ISSUE THREAD

**Issue by tehn** [monome/earthsea#2]
multi-press note priority. ie hold note A, then press note B. release note B and note A should re-trigger. same situation with arp transposition etc.

**Comment by ngwese** [comment]
I suspect this would be easy enough to implement using the [last note priority code](https://github.com/ngwese/mod/blob/520ecb90fcb4c7f0d8bc41ebc08f2099ca2e2ac2/skeleton/notes.c) added as part of the es midi changes.

I might give this a whirl once the reorg is done and the midi code merged into the mainline.
